style.css

```diff
@@ -82,76 +87,241 @@ a:hover {
   color: #e31c3a;
 }
 
-img {
-  width: 100%;
+.big-meal-photo {
+  width: 70%;
   height: auto;
+  margin: 0 auto;
 }
 
 .input-button-field {
   display: flex;
   flex-direction: column;
+  width: 100%;
+  margin: 0 auto;
 }
 
 .email {
-  margin-top: 2em;
+  margin-bottom: 1em;
   border: 1px solid #e31c3a;
-  border-radius: 4px;
+  border-radius: 0.25em;
   background-color: #ffe6ee;
   padding: 0.8em;
   color: #080808;
   outline: none;
 }
 
 .button {
-  margin: 1em 0;
+  margin-bottom: 1em;
   border: 1px solid #e31c3a;
-  border-radius: 4px;
+  border-radius: 0.25em;
   background-color: #e31c3a;
   padding: 0.8em;
   color: #ffffff;
   outline: none;
   cursor: pointer;
 }
 
+.section-two {
+  width: 100%;
+  padding: 0 1em;
+  margin-bottom: 3em;
+}
+
+.head-meal-section {
+  text-align: center;
+  margin-bottom: 0.5em;
+  line-height: 1em;
+  font-size: 2.2em;
+}
+
+.under-head-meal-section {
+  text-align: center;
+  font-size: 0.9em;
+  margin-bottom: 2em;
+}
+
+.about-you {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  margin-bottom: 2em;
+}
+
+.vege-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.rotateimg180 {
+  transform: rotate(90deg);
+}
+
+.title-about-you {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  text-align: center;
+  margin: 1em 0em;
+}
+
+.vege-text {
+  font-size: 0.75em;
+  margin-top: 1em;
+  text-align: center;
+}
+
+.meals {
+  display: flex;
+  flex-direction: column;
+}
+
+.refugee-block {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 2em;
+}
+
+.refugee-title {
+  color: #0a0a0a;
+  font-size: 2.2em;
+  text-align: center;
+  line-height: 1em;
+  margin-bottom: 1em;
+}
+
+.refugee-text {
+  font-size: 0.75em;
+  text-align: center;
+  margin-bottom: 1em;
+}
+
+.meal-block {
+  display: flex;
+  margin-bottom: 2em;
+}
+
+.meal-image {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.price-with-text {
+  display: flex;
+  flex-direction: column;
+  justify-content: left;
+  align-items: left;
+  margin: 0em 2em;
+}
+
+.price {
+  display: flex;
+  font-size: 2em;
+  font-weight: bold;
+}
+
+.dolar {
+  font-size: 0.7em;
+}
+
+.recipe-title {
+  font-size: 1.5em;
+  text-transform: uppercase;
+  margin-bottom: 0.5em;
+}
+
+.recipe-text {
+  font-size: 0.75em;
+}
+
+.section-three {
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto;
+  align-items: center;
+  background-color: #ffffff;
+  width: 100%;
+  padding: 0em 1em 3em 1em;
```

Comment:
Nitpicking, but this could be written shorter. (We have enough time, so let's learn shorthands also 😄 )